### DISCUSSION TITLE
How to use NavigationView in maui like WinUI3?

### DISCUSSION THREAD

**Discussion by stevexuzeta** [dotnet/maui#15699] [Q&A]
This [Navigation Document](https://learn.microsoft.com/en-us/windows/apps/design/controls/navigationview) shows how to use navigationview in winui3, how to use and show same display appearance in dotnet maui?

**Comment by mhrastegari** [comment]
This is what you need!
https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/shell/flyout

And BTW I think new MAUI templates by default have AppShell, but just in case take a look at this too:
https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/flyoutpage

**Reply by stevexuzeta** [reply]
Thanks for your answer. I wonder how to set open and close navigation button(icon with three short horizontal lines) in maui. When click open navigation button, it shows items with icons and titles. When click close navation button, it only shows items with icons. 
![1](https://github.com/dotnet/maui/assets/28527945/578b02a7-2588-480f-9456-b31a962c8be5)
![2](https://github.com/dotnet/maui/assets/28527945/9c208c78-c9b6-4928-bff2-0e46260e3e06)

**Reply by Mistofs** [reply]
what do you mean how to set
